django/db/models/sql/query.py

```diff
@@ -789,9 +784,9 @@ def change_aliases(self, change_map):
         # 1. Update references in "select" (normal columns plus aliases),
         # "group by" and "where".
         self.where.relabel_aliases(change_map)
-        if isinstance(self.group_by, list):
-            self.group_by = [col.relabeled_clone(change_map) for col in self.group_by]
-        self.select = [col.relabeled_clone(change_map) for col in self.select]
+        if isinstance(self.group_by, tuple):
+            self.group_by = tuple([col.relabeled_clone(change_map) for col in self.group_by])
```

Comment:
The list comprehension may be slightly more performant. I don't know if we care enough to profile it.